code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -164,19 +164,19 @@
 			var/mob/living/carbon/slime/S =  L
 			if(!status)
 				L.visible_message("<span class='warning'>[S] has been prodded with \the [src] by [user]. Too bad it was off.</span>")
-				return 1
+				return TRUE
 			else
 				L.visible_message("<span class='danger'>[S] has been prodded with \the [src] by [user]!</span>")
 
-			S.Discipline ++
-			if(prob(5))
-				S.Discipline = 0
-				S.rabid = 1 // heres that "or piss them off part"
+			S.discipline++
+			if(prob(1))
```

Comment:
Undocumented.

Reply:
`  - tweak: "Stunbatonning a slime now has a reduced chance of making them go batshit."`